curryst.typ

```diff
@@ -80,6 +80,13 @@
   /// even if some rules have no name (unless a rule is higher than a single
   /// line).
   min-bar-height: 0.8em,
+  /// Set the orientation of the proof tree to top to bottom instead of bottom
+  /// to top.
+  ///
+  /// If set to true, the conclusion will be at the top, and the premises will
+  /// be at the bottom. The proof tree will also be aligned to the top instead
+  /// of the bottom.
```

Comment:
```suggestion
  /// be at the bottom.
```

I don't think the alignment change is noticeable since `content` is wrapped in a box (line 394 in your PR).

Reply:
What I mean by this comment is that, by default, the proof tree's y-axis is aligned to the bottom, like that:

![bottom-aligned](https://github.com/user-attachments/assets/cad05c12-0a41-45cf-bf59-3176ffc37bbc)

And setting the `top-to-bottom` argument to `true` changes this behavior, because it wouldn't really make sense to build a top-to-bottom proof tree aligned at the bottom. This is addressed at lines 375 and 378.

![top-aligned](https://github.com/user-attachments/assets/e12c7d13-40a8-4a0a-a4bb-201d39dfde93)

But maybe the comment is not very well explained or even necessary.

Reply:
I don't think it is necessary, as this is probably what most people would expect anyway, because the first image doesn't look like anything people may want. I feel like adding this precision raises more questions than it answers (for example, I wondered if it had an effect regarding the alignment of the final tree, which it does not).